skimage/restoration/tests/test_denoise.py

```diff
@@ -1174,15 +1175,38 @@ def test_cycle_spinning_num_workers():
             max_shifts=1,
             func_kw=func_kw,
             channel_axis=None,
-            num_workers=4,
+            workers=4,
         )
         dn_cc3 = restoration.cycle_spin(
             noisy,
             denoise_func,
             max_shifts=1,
             func_kw=func_kw,
             channel_axis=None,
-            num_workers=None,
+            workers=None,
         )
     assert_array_almost_equal(dn_cc1, dn_cc2)
     assert_array_almost_equal(dn_cc1, dn_cc3)
+
+
+@xfail_without_pywt
+@pytest.mark.parametrize("num_workers", [None, 1])
```

Comment:
```suggestion
@pytest.mark.parametrize("num_workers", [1])
```
CI will pass with this change; I just tested locally, after uninstalling `dask`. I admit the syntax is a little overkill then, since you don't need `@pytest.mark.parametrize` to test for a single parameter value; but then we keep the spirit... Also, this test will go away once the deprecation cycle is over, so I wouldn't worry too much.

For reference, the current CI failure is: `FAILED scikit-image/restoration/tests/test_denoise.py::test_cycle_spinner_deprecate_num_workers[None] - UserWarning: The optional dask dependency is not installed. The number of workers is set to 1. To silence this warning, install dask or explicitly set `workers=1` when calling the `cycle_spin` function`

Reply:
I attempted a different fix https://github.com/scikit-image/scikit-image/pull/7302/commits/e23f5537c104240a9ea120c7aa722f58a7b73161 and fixed the stacklevel of the offending warning in the process. Should pass now. :crossed_fingers: